app/services/end_of_cycle_timetable.rb

```diff
@@ -1,85 +1,99 @@
 class EndOfCycleTimetable
-  CURRENT_YEAR_FOR_SCHEDULE = 2021
-
   # These dates are configuration for when the previous cycle ends and the next cycle starts
   # The 2020 dates are made up so we can generate sensible test data
   CYCLE_DATES = {
     2020 => {
-      apply_1_deadline: Date.new(2019, 8, 24),
-      stop_applications_to_unavailable_course_options: Date.new(2019, 9, 7),
-      apply_2_deadline: Date.new(2019, 9, 18),
-      find_closes: Date.new(2019, 10, 3),
       find_reopens: Date.new(2019, 10, 6),
       apply_reopens: Date.new(2019, 10, 13),
-    },
-    2021 => {
       apply_1_deadline: Date.new(2020, 8, 24),
       stop_applications_to_unavailable_course_options: Date.new(2020, 9, 7),
       apply_2_deadline: Date.new(2020, 9, 18),
       find_closes: Date.new(2020, 10, 3),
+    },
+    2021 => {
       find_reopens: Date.new(2020, 10, 6),
       apply_reopens: Date.new(2020, 10, 13),
+      apply_1_deadline: Date.new(2021, 8, 24),
+      stop_applications_to_unavailable_course_options: Date.new(2021, 9, 7),
+      apply_2_deadline: Date.new(2021, 9, 18),
+      find_closes: Date.new(2021, 10, 3),
     },
+    2022 => {
```

Comment:
There are sections of the code that need to check whether a particular date lies between and apply deadline and find / apply reopening. As a result we now need to 'look forward' to the next cycle to get these dates. Although these dates are not confirmed having them here will cause less test breakages. Updating these values if req'd should not affect the specs

Reply:
An alternative would be to create a new CYCLE_DATES hash specifically for testing and only have 'real' and known dates in the EndOfCycleTimetable class (ie not mixing with data that has been added for the purpose of testing, namely the 2020 and 2022 cycles)